src/routes/transactions/mappers/common/transaction-info.mapper.ts

```diff
@@ -342,17 +342,11 @@ export class MultisigTransactionInfoMapper {
     }
 
     try {
-      const isConfirmed =
-        'confirmations' in transaction &&
-        !!transaction.confirmations &&
-        transaction.confirmations.length >= transaction.confirmationsRequired;
```

Comment:
The general `txStatus` signifies if a transaction is awaiting signatures/executions. The `txInfo.status` of a staking transaction is inherently staking specific, e.g. `NOT_STAKED`, `DEPOSIT_IN_PROGRESS`, etc.